pyproject.toml

```diff
@@ -107,7 +107,22 @@ fix = true
 known-first-party = ["cellfinder_core"]
 
 [tool.mypy]
-python_version = "3.8"
```

Comment:
This is the Python version used to run typing. Hard coding this causes issues when other Python major versions are being used to run `mypy`, so I've removed it.